### PR TITLE
EASI-3835: remove logstash/opensearch from local deployment

### PR DESCRIPTION
**Pull request by Jdwoodson**
# EASI-3835

## Changes and Description

- remove logstash/opensearch from local deployment via docker compose

## PR Author Review Checklist

- [ ] Met the ticket's acceptance criteria, or will meet them in a subsequent PR.
- [ ] Added or updated tests for backend resolvers or other functions as needed.
- [ ] Added or updated client tests for new components, parent components, functions, or e2e tests as necessary.
- [ ] Updated the [Postman Collection](../MINT.postman_collection.json) if necessary.

## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.

### CODE REVIEW

**Comment by ClayBenson94** [comment]
@Jdwoodson Converting this to draft. I think we should have the backend code removed on this as well. I'll work with the BE folks to get this done in this PR! (That way we can tag the commit in `main` before this goes in as the final "pre removal" commit)

File: .spelling [hunk 1]
```diff
@@ -219,10 +219,6 @@ routes.go
 inlining
 Performant
 serializable
-OpenSearch
-Opensearch
```
Comment (ClayBenson94) [inline .spelling:223]: Removing these lines is causing linters to fail due to these terms existing in ADRs. These shouldn't be removed for that reason!

**Review by ClayBenson94 — CHANGES_REQUESTED** [review]
Also looks like the `pkg/querybuilderfilters` tests are failing due to missing references to auto-generated types no longer being present.

I think we should just delete the `querybuilderfactories`, `querybuilderfilters`, and `pkg/oddsearch` packages along with the other code here, since it all ties in with OpenSearch.

**Comment by OddTomBrooks** [comment]
Since we have review on this I’m promoting this from draft and addressing feedback.

**Review by ClayBenson94 — APPROVED** [review]
Changes look good! @Jdwoodson, since you're the author, are you good to merge this and deploy it out?

**Pull request merged**